source/mind_ir.js

```diff
@@ -0,0 +1,755 @@
+/**
+ * @fileoverview
+ * This file contains the implementation of the mind_ir namespace, including the ModelFactory, Model, and Graph classes.
+ * The ModelFactory class is responsible for matching and opening mind_ir models.
+ * The Model class represents a mind_ir model and contains information about the model's format, name, and graphs.
+ * The Graph class represents a graph within a mind_ir model and contains information about the graph's inputs, outputs, and nodes.
+ */
+export const protobuf = {};
+const mind_ir = {
```

Comment:
* Rename to `const mindir = {};`.
* Follow pattern in other files that do not predefine empty classes.